### PR TITLE
Feat/followers

### PR DESCRIPTION
**Pull request by MacenaAndre**
- post /follow service function
- get /following service function
- button behavior
- Api requests

### CODE REVIEW

**Review by guilhermeicarofr — APPROVED** [review]
Very good

File: src/components/follows/followButton.js [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
 
     const [disableFollow, setDisableFollow] = useState(false)
     const [following, setFollowing] = useState(false)
+    const { config } = useContext(LoginContext);
+
+    useEffect(() => {
+        isFollowing({config, followedId})
+            .then((res) => {
+                console.log(following);
+                setFollowing(res.data.isFollowing);
```
Comment (tissyane) [inline src/components/follows/followButton.js]: Remove this console.log before merging

File: src/components/follows/followButton.js [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+            })
+            .catch((error) => {
+                console.log(error);
+            })
```
Comment (tissyane) [inline src/components/follows/followButton.js]: Is this console.log necessary?

**Review by tissyane — APPROVED** [review]

**Comment by MacenaAndre** [comment]
Thanks for the heads ups!!

**Pull request merged**